cli/uninstall/uninstall.go

```diff
@@ -82,3 +83,32 @@ func UninstallProgram(program string, binDst string, headerDst string,
 	log.Infof("%s is uninstalled.", program)
 	return err
 }
+
+// GetList generates a list of options to uninstall.
+func GetList(cmdCtx *cmdcontext.CmdCtx) []string {
```

Comment:
Add a unit-test for this public function.

Reply:
Added